docs/spark/how-to-guides/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Deploy Worker and UDF Binaries
+description: Learn how to deploy Worker and UDF (User-Defined Function) binaries.
+ms.date: 12/30/2019
```

Comment:
update date

Reply:
May I know which date I should use? I put 12/30/2019 since that's the date when I created the PR.